### ISSUE TITLE
Update Home Page Colors

### ISSUE THREAD

**Issue by covjos** [jclark2121/WebTechProject#8]
I like all of the content you used throughout the site it is all very good. One thing that I did notice that could be fixed is that the home page is really dark and uninviting in my opinion but the rest of the page is light and more inviting , so I would say to change the home page color to being maybe a white background or something instead of a black background would make it a lot more inviting for users. Maybe if you can add a cool animation too to the home page to give it more of an intrigue would help a lot too. 
![image](https://user-images.githubusercontent.com/122822195/229564767-9eca4334-3251-4333-a10f-d51090173f32.png)

**Comment by jclark2121** [comment]
I agree, I think the dark contrasts with the rest of the site too much
